geowebcache/rest/src/test/resources/org/geowebcache/rest/webresources/test.js

```diff
@@ -0,0 +1 @@
+JS
```

Comment:
This seems strange, does it really need to have content?

Reply:
Not really but the others had it so I included it for consistency.  I think when I originally created these tests I was originally planning to check that the content was included but never added that assertion.

Reply:
Fair enough